.github/workflows/test_and_docs.yml

```diff
@@ -0,0 +1,58 @@
+# *******************************************************************************
+# Copyright (c) 2025 Contributors to the Eclipse Foundation
+#
+# See the NOTICE file(s) distributed with this work for additional
+# information regarding copyright ownership.
+#
+# This program and the accompanying materials are made available under the
+# terms of the Apache License Version 2.0 which is available at
+# https://www.apache.org/licenses/LICENSE-2.0
+#
+# SPDX-License-Identifier: Apache-2.0
+# *******************************************************************************
+
+name: Tests, Verify and Build Docs
+
+permissions:
+  contents: write
+  pages: write
+  pull-requests: write
+  id-token: write
+
+on:
+  pull_request_target:
+    types: [opened, reopened, synchronize] # Allows forks to trigger the docs build
+  push:
+    branches:
+      - main
+  merge_group:
+    types: [checks_requested]
+
+jobs:
+  docs-verify:
+    uses: eclipse-score/cicd-workflows/.github/workflows/docs-verify.yml@main
+    permissions:
+      pull-requests: write
+      contents: read
+    with:
+      bazel-docs-verify-target: "//:docs_check"
+
+  # This is the user configurable part of the workflow
+  consumer-tests:
+    uses: ./.github/workflows/consumer_test.yml
+    secrets: inherit
+
+  docs-build:
+    # Waits for consumer-tests but run only when docs verification succeeded
+    needs: [docs-verify, consumer-tests]
+    if: ${{ always() && needs.docs-verify.result == 'success' }}
+    uses: eclipse-score/cicd-workflows/.github/workflows/docs.yml@main
+    permissions:
+      contents: write
+      pages: write
+      pull-requests: write
+      id-token: write
+    with:
+      bazel-target: "//:docs -- --github_user=${{ github.repository_owner }} --github_repo=${{ github.event.repository.name }}"
+      retention-days: 3
+      tests-report-artifact: tests-report
```

Comment:
Seems random, that consumer-tests upload into that exact name.

How about `consumer-tests.output.test_results`?

Reply:
(Can also be adjusted in separate PR)